content/influxdb/cloud-dedicated/reference/internals/security.md

```diff
@@ -222,12 +237,78 @@ Users can configure the following security controls:
 
 ### Access, authentication, and authorization
 
-We use [Auth0](https://auth0.com/) for InfluxDB Cloud Dedicated authentication.
-User accounts can be created by InfluxData on the InfluxDB Cloud Dedicated system via Auth0.
-User accounts can create database tokens with data read and/or write permissions.
-API requests from custom applications require a database token with sufficient permissions.
-For more information on the types of tokens and ways to create them, see
-[Manage tokens](https://docs.influxdata.com/influxdb/cloud-dedicated/admin/tokens/).
+InfluxDB Cloud Dedicated uses [Auth0](https://auth0.com/) for authentication and separates workload cluster management authorizations (using _management tokens_) from database read and write authorizations (using _database tokens_).
+
+- [User provisioning](#user-provisioning)
+- [Management tokens](#management-tokens)
+- [Database tokens](#database-tokens)
+
+#### User provisioning
+
+InfluxData creates user accounts on the InfluxDB Cloud Dedicated system via Auth0.
+After a user account is created, InfluxData provides the user with the following:
+
+- An **Auth0 login** to authenticate access to the cluster
+- The InfluxDB Cloud Dedicated **account ID**
+- The InfluxDB Cloud Dedicated **cluster ID**
+- The InfluxDB Cloud Dedicated **cluster URL**
+- A password reset email for setting the login password
+
+With a valid password, the user can login via InfluxData's `influxctl` command line tool.
+The login command initiates an Auth0 browser login so that the password is never exchanged with `influxctl`.
+With a successful authentication to Auth0, InfluxDB Cloud Dedicated provides the user's `influxctl` session with a short-lived [management token](#management-tokens) for access to the Granite service.
+The user interacts with the `influxctl` command line tool to manage the workload cluster, including creating [database tokens](#database-tokens) for database read and write access.
+
+#### Management tokens
+
+Management tokens authenticate user accounts to the Granite service and provide authorizations for workload cluster management activities, including:
+ 
+ - account, pricing, and infrastructure management
+ - inviting, listing, and deleting users
+ - creating, listing, and deleting, and databases
+ - creating, listing, and deleting database tokens
+
+Management tokens consist of the following:
+
+- An access token string (sensitive)
+- A permissions set for management activities (configured by InfluxData during initial provisioning)
```

Comment:
It might be more clear to say 'configured when provisioning a user' since that will cover the first user that is initially provisioned, but also newly invited users. This language should also be general enough to cover the future fine-grained management token permissions when creating a new user.

Reply:
@jdstrand 
> It might be more clear to say 'configured when provisioning a user' since that will cover the first user that is initially provisioned, but also newly invited users. This language should also be general enough to cover the future fine-grained management token permissions when creating a new user.

Is the permission set associated with the account, i.e., Is it accurate to say this:

```
InfluxData creates user accounts and assigns permissions to user accounts on the InfluxDB Cloud Dedicated system via Auth0.
```

Reply:
Also, should I add something like "authorized users can invite new users and set permissions for new user accounts"?

Reply:
This is accurate: "InfluxData creates user accounts and assigns permissions to user accounts on the InfluxDB Cloud Dedicated system via Auth0."

> Also, should I add something like "authorized users can invite new users and set permissions for new user accounts"?

No, not yet as we don't yet have fine-grained management token permissions exposed to users.